src/components/organisms/Content/Content.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Router } from '@reach/router';
+
+//For tests. Delete it when you will need.
+const Test = () => <p>content</p>;
+
+function Content() {
+    return (
+        <Router className="content">
```

Comment:
Routing powinien być z założenia na najwyższym poziomie aplikacji, czyli to powinno wylądować w naszym przypadku w `App.js`